### DISCUSSION TITLE
Brand new. Interested in creating new languages.

### DISCUSSION THREAD

**Discussion by VLCampbell** [espeak-ng/espeak-ng#2046] [General]
Hi. I'm brand new here and please forgive my exceedingly ignorant questions. 

I'm interested in learning how to create a voice or voices for Scottish Gaelic. I have no idea where to start and this place seems overwhelming. Can someone please point me in the direction of where to find my first steps? 

Thank you for any help you can provide on getting started.

**Comment by isolveit-aps** [comment]
Hi VLCampbell, sorry for the slow followup. I just recently started contributing myself. 
The most hands-on description is in this doc: https://github.com/espeak-ng/espeak-ng/blob/master/docs/add_language.md
But please also read the doc : https://github.com/espeak-ng/espeak-ng/blob/master/docs/contributing.md

**Reply by VLCampbell** [reply]
Thank you! I'll have a look!
